Add install-deps key to cljs_options

ajpierce opened the pull request:
As of Clojurescript 1.9.854, the compiler no longer automatically downloads javascript dependencies defined in `:npm-deps`.  However, the Clojurescript team [added](https://github.com/clojure/clojurescript/commit/fb8ce05143dac9e9feb602be2544b72c87b337a3) a new flag to the compiler, `:install-deps`, that, when set to true, will download the JS dependencies for you.

This PR adds the `:install-deps` key to `cljs_options` to allow users to pass this flag through to the compiler for automatic JS dependency downloading.

Thanks for all your hard work, Bruce!  Let me know if there's anything you'd like me to change or add to this PR :)

bhauman commented:
This is a good start. Do a search for `::npm-deps` in the file to find the other reference as an example of what else is needed here.

ajpierce commented:
Ok, I've added the missing key to the compiler options, sorry about that!  I went ahead and rebased the commits so as not to clutter up the commit history.

Let me know if there's anything else I'm missing, and thanks for the help!

bhauman commented:
Thanks!!
